CMakeLists.txt

```diff
@@ -116,6 +116,11 @@ if (${CMAKE_VERSION} VERSION_LESS "3.6.0")
     set(QUIC_ENABLE_LOGGING OFF)
 endif()
 
+if(QUIC_PLATFORM STREQUAL "darwin)
+    message(WARNING "macOS does not support logging currently")
+    set(QUIC_ENABLE_LOGGING OFF)
```

Comment:
Why not? CLOG was running before this.

Reply:
CLOG was, but we were always passing DisableLogs because lttng isn't supported.

Reply:
But I can fix that a different way